packages/store/internals/testing/src/fresh-platform.ts

```diff
@@ -30,11 +30,26 @@ function resetPlatformAfterBootstrapping() {
   createPlatform(TestBed);
 }
 
-export function freshPlatform(fn: Function): (...args: any[]) => any {
-  return async function testWithAFreshPlatform(this: any, ...args: any[]) {
+export function freshPlatform(fn: (done?: VoidFunction) => Promise<void>) {
+  let done: VoidFunction | null = null,
```

Comment:
Technically, all these variants do the same thing. So, it’s a matter of personal taste and aesthetics.
The multiline variant is a bit longer, but easier to read:

```ts
let done: VoidFunction | null = null;
let whenDoneIsCalledPromise: Promise<void> | null = null;
```

Reply:
"easier to read" is also a personal taste :wink:

Reply:
up to you)